python/cudf_polars/tests/experimental/test_groupby.py

```diff
@@ -10,15 +10,6 @@
 from cudf_polars.testing.asserts import assert_gpu_result_equal
 
 
-@pytest.fixture(scope="module")
-def engine():
```

Comment:
Should we also remove the redundant `engine` definitions from `test_shuffle.py` and `test_select.py`?